coin_chk_libhdr.m4

```diff
@@ -354,6 +363,22 @@ dnl to do the heavy lifting.
     AC_MSG_RESULT([$m4_tolower(coin_has_$1) (COIN_SKIP_PROJECTS)])
   fi
 
+dnl Try to offer some helpful advice in the event of failure.
+  if test "$m4_tolower(coin_has_$1)" = "no" ; then
+    if expr "$m4_tolower($1_failmode)" : '.*header.*' &>/dev/null ; then
+      AC_MSG_WARN(m4_normalize(
+	[Compiler flags are "$m4_tolower($1)_cflags".]
+	[Check that they are correct.]
+	[You can supply correct values using --with-m4_tolower($1)-cflags.]))
+    fi
+    if expr "$m4_tolower($1_failmode)" : '.*link.*' &>/dev/null ; then
+      AC_MSG_WARN(m4_normalize(
+	[Linker flags are "$m4_tolower($1)_lflags".]
+	[Check that they are correct.]
+	[You can supply correct values using --with-m4_tolower($1)-lflags.]))
+    fi
+  fi
+
```

Comment:
Helpful advice. A bit fragile --- if the failure modes set in FIND_PRIM_LIBHDR change, this can break.

Reply:
Users often don't recall that they should look into config.log regarding details why a compile or link check failed. Maybe a hint towards config.log could be added here, too.

Reply:
Will do.